### PR TITLE
Translate 2016 04 07 react v15

### LINKED ISSUE #284: Translate 2016-04-07-react-v15.md

### PR DESCRIPTION
**Pull request by elderferreiras**
Started the translation of the article "React v15.0".

close #284

### CODE REVIEW

**Comment by elderferreiras** [comment]
I have finished this translation. I went through the final text to make sure I was following the standards set up [here](https://github.com/reactjs/pt-BR.reactjs.org/blob/master/GLOSSARY.md). I also made some fixes regarding wrong `link` definitions. I'm happy to help with other translations when this one gets reviewed.

File: content/blog/2016-04-07-react-v15.md [hunk 1]
```diff
@@ -3,257 +3,256 @@ title: "React v15.0"
...
 
-* We received some amazing contributions from the community in this release, and we would like to highlight [this pull request](https://github.com/facebook/react/pull/5753) by [Michael Wiencek](https://github.com/mwiencek) in particular. Thanks to Michael’s work, React 15 no longer emits extra `<span>` nodes around the text, making the DOM output much cleaner. This was a longstanding annoyance for React users so it’s exciting to accept this as an outside contribution.
+* Recebemos algumas contribuições surpreendentes da comunidade nessa release e gostaríamos de destacar em particular [essa pull request](https://github.com/facebook/react/pull/5753) do [Michael Wiencek](https://github.com/mwiencek). Graças ao trabalho do Michael, o React 15 não emite mais extra `<span>` nós ao redor do texto, tornando o resultado do DOM muito mais limpo. Esse era um incômodo antigo dos usuários do React, por isso é tão legal que essa mudanca seja uma contribuição externa.
```
Comment (jussarasoares) [inline content/blog/2016-04-07-react-v15.md]:
```suggestion
* Recebemos algumas contribuições surpreendentes da comunidade nessa release e gostaríamos de destacar em particular [essa pull request](https://github.com/facebook/react/pull/5753) do [Michael Wiencek](https://github.com/mwiencek). Graças ao trabalho do Michael, o React 15 não emite mais extra `<span>` nós ao redor do texto, tornando o resultado do DOM muito mais limpo. Esse era um incômodo antigo dos usuários do React, por isso é tão legal que essa mudança seja uma contribuição externa.
```

File: content/blog/2016-04-07-react-v15.md [hunk 2]
```diff
@@ -3,257 +3,256 @@ title: "React v15.0"
...
...id` para cada nó do DOM. Embora essa mudança torne mais difícil saber se um site está usando o React, a vantagem é que o DOM fica muito mais leve. Essa alteração foi possível ao escolhermos o uso de `document.createElement` na renderização inicial. Anteriormente, gerávamos uma grande string de HTML e, em seguida, definíamos `node.innerHTML`. Na época, decidiu-se que essa abordagem era mais rápida do que usar `document.createElement` para a maioria dos casos e navegadores compatíveis. Mas os navegadores continuaram a melhorar tanto que a razão por trás da nossa abordagem inicial não é mais verdadeira. Ao usar `createElement`, podemos fazer com que outras partes do React fiquem mais rápidas. Os IDs foram usados para mapear os eventos para o componente React original, o que significa que tínhamos que fazer muita coisa em todos os eventos, mesmo já tendo esses dados armazenados em cache. Como todos já vimos, o armazenamento em cache e, em particular, a invalidação de caches, podem ser propensos a erros e, como resultado, vimos muitos problemas difíceis de serem reproduzidos. Agora podemos construir um mapeamento direto no momento da renderização, já que já temos um identificador no nó.
```
Comment (jussarasoares) [inline content/blog/2016-04-07-react-v15.md]:
```suggestion
    Aconteceram várias mudanças grandes na forma como interagimos com o DOM. Uma das mudanças mais notáveis é que não definimos mais o atributo `data-reactid` para cada nó do DOM. Embora essa mudança torne mais difícil saber se um site está usando o React, a vantagem é que o DOM fica muito mais leve. Essa alteração foi possível ao escolhermos o uso de `document.createElement` na renderização inicial. Anteriormente, gerávamos uma grande string de HTML e, em seguida, definíamos `node.innerHTML`. Na época, decidiu-se que essa abordagem era mais rápida do que usar `document.createElement` para a maioria dos casos e navegadores compatíveis. Mas os navegadores continuaram a melhorar tanto que a razão por trás da nossa abordagem inicial não é mais verdadeira. Ao usar `createElement`, podemos fazer com que outras partes do React fiquem mais rápidas. Os IDs foram usados para mapear os eventos para o componente React original, o que significa que tínhamos que fazer muita coisa em todos os eventos, mesmo já tendo esses dados armazenados em cache. Como todos já vimos, o armazenamento em cache e, em particular, a invalidação de caches, podem ser propensos a erros e, como resultado, vimos muitos problemas difíceis de serem reproduzidos. Agora podemos construir um mapeamento direto no momento da renderização, já que já temos um identificador no nó.
```

**Comment by elderferreiras** [comment]
Thank you @jussarasoares for taking the time to review my translation. I've updated the pull request with your suggestions.

**Pull request merged**